_data/simple-icons.json

```diff
@@ -270,6 +310,11 @@
             "hex": "EE6123",
             "source": "https://bitly.com/pages/press"
         },
+        {
+            "title": "Blackberry",
+            "hex": "00BFFF",
+            "source": "https://www.blackberry.com/content/dam/blackberry-com/Documents/pdf/legal/blackberry-branding-guidelines-external.pdf"
```

Comment:
I would probably prefer just using https://www.blackberry.com/en. The logo is available as an SVG in their header, and it would allow us to check for updates (contrary to PDFs that might not be updated)